code/headers/hx711.hpp

```diff
@@ -0,0 +1,41 @@
+// #pragma once
```

Comment:
This file has no doxygen. I think it would be a good idea to add doxygen to this header, to make it clear what the classes are all about and what the functions are for.

Reply:
Code looks good, perhaps delete the redundant comments in your code.

Reply:
Yes, documentation would make your code more easily understood.

Reply:
I've added documetation